types/ext/backend.d.ts

```diff
@@ -31,3 +31,12 @@ export type TabInfo = {
 };
 
 export type FindTabsPredicate = (tabInfo: TabInfo) => boolean | Promise<boolean>;
+
```

Comment:
May not want this to be an enumeration, as I believe these values come from `Error`s thrown by the browser, which basically have arbitrary strings.